source/stormwater.rst

```diff
@@ -0,0 +1,329 @@
+
+Calculate water quality benefits of stormwater retention (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The potential water quality impact of stormwater retention is determined as the pollutant mass associated with retained stormwater, i.e. the amount of pollutant load avoided. The annual avoided pollutant load and, in :math:`kg/yr`, is calculated for each pixel :math:`i` as the product of retained volume (:math:`m3/yr`) and the event mean concentration (EMC) of a pollutant, in :math:`mg/L`:
+
+.. math:: Avoided.load_i=0.001\cdot V_{RE,i}\cdot EMC
+
+Similarly, the annual pollutant load (:math:`kg/yr`) is calculated from the runoff volume:
+
+.. math:: Load_i=0.001\cdot V_{RU,i}\cdot EMC
+
+
+EMCs for each pollutant assigned to land use classes using the biophysical table. Nitrogen and phosphorus are common pollutants of interest, but any stormwater pollutants (such as sediment, metals, or organic compounds) may be used by providing EMC values for those pollutants in the biophysical table. If no pollutants are included, this step is skipped.
+
+Valuation of stormwater retention service (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A review of the most common valuation methods for the stormwater retention service can be found in a working paper by Sahl et al. (2015). If stormwater regulations exist, the user can assess the value of stormwater retention with the target retention volume as a reference. The economic value can be assessed if the average value of retention device (:math:`$/m^3`) is available:
+
+.. math:: Retention.cost=PR\cdot VRE
+   :label: retention-value
+
+where :math:`PR` is the replacement cost of stormwater retention (:math:`$/m^3`). For example, Simpson and McPherson (2007) estimate this to be :math:`$1.59/m^3` for urban areas in the San Francisco Bay area.
+
+The model can output potential groundwater recharge volume :math:`VI`, which may also serve as a valuation of retention. However, the model does not currently estimate the pollutant load associated with this recharge volume, as sub-surface transport and transformation of pollutants is not implemented in the model.
+
+Aggregation at the watershed scale (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users may provide a polygon vector file outlining areas over which to aggregate data (typically watersheds or sewersheds). The model will aggregate the output rasters to compute:
+
+- Average stormwater retention ratio (average of :math:`RE` values)
+- Total retention volume, :math:`m^3` (sum of :math:`VRE` values)
+- Total retained pollutant load for each pollutant, :math:`kg/yr` (sum of :math:`Avoided.load` values)
+- Total pollutant load for each pollutant, :math:`kg/yr` (sum of :math:`Load` values)
+- Total potential recharge volume, :math:`m^3` (sum of :math:`VI`, if infiltration data provided)
+- Total Replacement Cost, currency units (sum of retention costs, if value data provided)
+
+
```

Comment:
Would be nice to get a "Limitations and Simplifications" section in here like we have for other models. Specifically this could address the seemingly high valuation outputs, which I think Ben said could be because most reporting uses the export portion.